include/swift/Runtime/Config.h

```diff
@@ -176,15 +176,17 @@ extern uintptr_t __COMPATIBILITY_LIBRARIES_CANNOT_CHECK_THE_IS_SWIFT_BIT_DIRECTL
 #define SWIFT_INDIRECT_RESULT
 #endif
 
-#if __has_attribute(swift_async_context)
+#if __has_feature(swiftasynccc) && USE_SWIFT_ASYNC_LOWERING && \
```

Comment:
Please remove this part. We can always use `swift_async_context`. We do so today.

Reply:
Removed.